### ISSUE TITLE
Possible confusing or incorrect documentation in `jj unsquash`.

### ISSUE THREAD

**Issue by askreet** [martinvonz/jj#3432]
## Description

This may well be me failing to understand the wording here, but I just used `jj unsquash -i` for the first time. Super useful, but I _think_ the documentation is incorrect, or the use of the difftool is inverted. I'm using the builtin tool.

The docs read:

> Move changes from a revision's parent into the revision

>   -i, --interactive
          Interactively choose which parts to unsquash

If we're defining the act of unsquashing as the moving of diff hunks from the current revisions parent into the current revision, then the things I select ("interactively choose") in the TUI should be the things that move, but the inverse is true. At least when I used this just now, the things I selected remained in the parent revision, while the rest was "unsquashed". (`jj undo` makes this type of experimentation easy 😄)

## Specifications

- Platform: OS X Ventura
- Version: 0.13.0

**Comment by joyously** [comment]
not latest release?

**Comment by askreet** [comment]
Thanks for the callout ,@joyously. I just updated to v0.15.1. It seems like the documentation is roughly the same, as is the behavior.

**Comment by arxanas** [comment]
I also opened https://github.com/arxanas/scm-record/issues/50 to track related issues.

**Comment by steveklabnik** [comment]
`jj squash` has basically the same text today, but since it's evolved into a more generic "move diff from here to there" command rather than specifically talking about the parent into the current revision, I think this text is okay today. If anyone disagrees please let me know!

**Comment by askreet** [comment]
@steveklabnik Makes sense to me. I think removing `unsquash` and adding explicit `--from` and `--to` make it very clear what I'm selecting. 👍🏼
